### ISSUE TITLE
Clarification Required on find() Access Time in Exercises 1.5.4 and 1.5.3

### ISSUE THREAD

**Issue by xzy2022** [reneargento/algorithms-sedgewick-wayne#307]
In Exercise 1.5.4, it's noted that executing the find() function on a node at depth i results in i+1 accesses to the id[] array. This was particularly apparent when analyzing the example with the second pair input, 3 8, where executing find(3) resulted in two accesses since the parent of node 3 is node 4. However, there seems to be a contrasting scenario in Exercise 1.5.3, where the find() function on a node at the same depth i necessitates 2i+1 accesses.

This variance has caused some uncertainty, and I'm inclined to believe the formula from Exercise 1.5.3 might be the correct interpretation. Could there be a detail I've missed that explains this inconsistency, or is there a definitive correct approach among the two? I am seeking further clarification to comprehend the intended behavior of the find() function within these contexts

**Comment by reneargento** [comment]
You are correct, in the `find()` method we should count `2i + 1` accesses on depth `i`.
I updated exercise 1.5.4 to fix that: https://github.com/reneargento/algorithms-sedgewick-wayne/commit/62da660403de4556f1ec0462e3847716a83293d0
Thanks for reporting this issue and for the contribution.
